### PR TITLE
Improve error checking code and fix sql script

### PR DESCRIPTION
**Pull request by javfg**

### CODE REVIEW

File: pkg/notification/db_changes.sql [hunk 1]
```diff
@@ -41,11 +41,12 @@ COMMIT;
 CREATE INDEX `cbox_notifications_ix0` ON `cbox_notifications` (`ref`);
 
 CREATE INDEX `cbox_notification_recipients_ix0` ON `cbox_notification_recipients` (`notification_id`);
-CREATE INDEX `cbox_notification_recipients_ix1` ON `cbox_notification_recipients` (`user_name`);
+CREATE INDEX `cbox_notification_recipients_ix1` ON `cbox_notification_recipients` (`recipient`);
 
 -- changes for added notifications on oc shares
 
 ALTER TABLE cernboxngcopy.oc_share ADD notify_uploads BOOL DEFAULT false;
```
Comment (glpatcern) [inline pkg/notification/db_changes.sql:48]: Can you remove the reference to the db name (`cernboxngcopy`) so we make the script generic?

**Review by glpatcern — CHANGES_REQUESTED** [review]

File: pkg/notification/db_changes.sql (hunk 1 above)
Comment (glpatcern) [reply pkg/notification/db_changes.sql:48]: (that was my comment as well when we reviewed the main PR. Maybe time to fix this before we go to prod, as then it's going to be impossible? :) )

Not that we properly respected a naming convention, but the tables can just be created without any prefix.

Comment (javfg) [reply pkg/notification/db_changes.sql:48]: Done!

File: pkg/notification/db_changes.sql [hunk 2]
```diff
@@ -19,33 +19,34 @@
...
 
-CREATE INDEX `cbox_notification_recipients_ix0` ON `cbox_notification_recipients` (`notification_id`);
-CREATE INDEX `cbox_notification_recipients_ix1` ON `cbox_notification_recipients` (`user_name`);
+CREATE INDEX `notification_recipients_ix0` ON `notification_recipients` (`notification_id`);
+CREATE INDEX `notification_recipients_ix1` ON `notification_recipients` (`recipient`);
 
 -- changes for added notifications on oc shares
 
 ALTER TABLE cernboxngcopy.oc_share ADD notify_uploads BOOL DEFAULT false;
+ALTER TABLE cernboxngcopy.oc_share ADD notify_uploads_extra_recipients VARCHAR(2048);
```
Comment (glpatcern) [inline pkg/notification/db_changes.sql:49]: Great! Still this one refers to the DB name...

Comment (javfg) [reply pkg/notification/db_changes.sql:49]: Fixed. As long as the script is running with the proper database selected beforehand, that shouldn't be needed.

Comment (glpatcern) [reply pkg/notification/db_changes.sql:49]: That's correct, and the script must be generic - the choice of the db is for the person deploying the script!
But.. did you push? :)

Comment (javfg) [reply pkg/notification/db_changes.sql]: I did, but to the test branch :laughing:  Pushed here now.

**Review by gmgigi96 — APPROVED** [review]

**Review by glpatcern — APPROVED** [review]
Modulo the actual schema change on the `oc_share` we quickly discussed, this is so far OK

**Pull request merged**